apps/src/lib/config/genesis.rs

```diff
@@ -372,6 +372,7 @@ pub mod genesis_config {
                         "Max commission rate change must be between 0.0 and \
                          1.0",
                     ),
+                metadata: Default::default(),
```

Comment:
we should add the metadata to `ValidatorConfig` and pass them through here to allow to set these at genesis

Reply:
done in d356f824843c2a50089651d773240c0d2b7aa077